hfc/fabric_ca/caservice.py

```diff
@@ -238,7 +238,15 @@ def generateAuthToken(self, req, registrar):
         else:
             bodyAndCert = b'.%s' % b64Cert
 
-        sig = self._cryptoPrimitives.sign(registrar._private_key, bodyAndCert)
+        # Construct the string to be signed: HTTP method + path + body + client cert
+        string_to_sign = {
+                            "http_method": "POST",
+                            "path": self._base_url,
+                            "body": b64Body.decode(),
+                            "client_cert": b64Cert.decode()
```

Comment:
@MuthuSundaravadivel 
I believe you need to use the base64 encoded values, and use a period (.) as a separator. See the other PRs for the exact format.

Also, does Python SDK only do POSTs? The other clients also had GETs and DELETEs I believe.